### PR TITLE
New about page design with photos and partners

### PR DESCRIPTION
**Pull request by schlosser**

### CODE REVIEW

File: src/components/StapledSidebar/StapledSidebar.style.tsx [hunk 1]
```diff
@@ -50,12 +59,27 @@ export const StyledSidebarLink = styled.a`
...
+  padding-top: ${NAV_HEIGHT + INNER_PADDING - 1}px;
+  margin-top: -${NAV_HEIGHT - OUTER_PADDING}px;
+
+  &:before {
+    display: block;
+    width: 100%;
+    content: ' ';
+    transform: translateY(-${INNER_PADDING}px);
+    border-top: 1px solid rgba(0, 0, 0, 0.12);
+  }
```
Comment (mikelehen) [inline src/components/StapledSidebar/StapledSidebar.style.tsx:78]: Wowsers...  I'm not sure exactly what's going on here (my CSS is weak), but it seems pretty complicated for a section heading...  Consider adding a comment (optional)?

File: src/screens/HomePage/HomePage.js [hunk 2]
```diff
@@ -21,9 +19,8 @@ export default function HomePage() {
       <main>
         <div className="App">
           <Content>
-            {!isMobile && <CriteriaExplanation />}
+            <CriteriaExplanation />
```
Comment (mikelehen) [inline src/screens/HomePage/HomePage.js:22]: I assume this change is intentional, but it wasn't mentioned in the PR description so please confirm.

Comment (schlosser) [reply src/screens/HomePage/HomePage.js:22]: Yep – we show above the map in all cases now. Before it was top on desktop only.

File: src/components/StapledSidebar/StapledSidebar.style.tsx (hunk 1 above)
Comment (schlosser) [reply src/components/StapledSidebar/StapledSidebar.style.tsx:79]: Comment added! this positions the scroll-to links perfectly aligned with the headers.

**Pull request merged**